pilot/pkg/bootstrap/server.go

```diff
@@ -84,6 +86,7 @@ const (
 )
 
 func init() {
+	startupTime = time.Now()
```

Comment:
It's actually pretty important, I can add it in a separate PR - we want to log how fast the port is up, after sync.

Reply:
Any reason to not keep track of the startup time ? Is it because the log is missing ? I have it in a separate PR that generates a K8S event including startup time.

Reply:
So can let it in

Reply:
ok